### DISCUSSION TITLE
Use absolute path rather than relative path in rss feed

### DISCUSSION THREAD

**Discussion by Inerska** [statiqdev/Discussions#159] [Q&A]
Hey, looking for a way to display the absolute link in my RSS feed ;

I'm using an action that displays the latest blog posts on my GitHub readme page.
https://agdl.dev/feed.rss

https://github.com/inerska/inerska

You could see that it's working but it's the relative path so when you click on a link it doesn't redirect you to the specified blog article.

Thanks in advance.
BTW: I'm taking any review or advice considered to my blog propulsed by statiq.web

**Comment by Inerska** [comment]
bump

**Comment by StevenTCramer** [comment]
Can you give us a little more information?  I am guessing (really) that where you generate your Link you are using Document.GetLink and you may want to use Context.GetLink.  But I don't see the statiq code you are using from the above links.

**Reply by Inerska** [reply]
My static blog is from this repository = 
https://github.com/inerska/inerska.github.io

https://github.com/Inerska/inerska.github.io/blob/master/MyBlog/input/_partials/_post.cshtml
And that's the file containing how a post feed should be display in my website : agdl.dev
(There are the green rectangle when hover)

**Comment by Inerska** [comment]
Sorry to bump again but that's a really annoying issue.
